tests/test_ProblemData.py

```diff
@@ -1314,6 +1314,49 @@ def test_max_lateness(solver: str):
     assert_equal(result.best.tasks[1].end, 2)
 
 
+def test_total_setup_time(solver: str):
```

Comment:
```suggestion
def test_total_setup_time(): # TODO implement for CP Optimizer
```

Combined with the comment below, this will make the CI pass successfully for now :)